src/nvapi.cpp

```diff
@@ -7,6 +7,8 @@
 #include "util/util_log.h"
 #include "../version.h"
 
+#include <mutex>
```

Comment:
Could you please move that include into nvapi_private.

Reply:
Done.